parser/routing.go

```diff
@@ -0,0 +1,97 @@
+package parser
+
+import (
+	"github.com/deis/workflow-cli/cmd"
+	docopt "github.com/docopt/docopt-go"
+)
+
+// Routing displays all relevant commands for `deis routing`.
+func Routing(argv []string) error {
+	usage := `
+Valid commands for routing:
+
+routing:list       list routability for an app
```

Comment:
I find list a `list` a little confusing, because it implies a list. Would `info` or `get` possibly make more sense?